cfgov/unprocessed/js/routes/prepaid-agreements/single.js

```diff
@@ -0,0 +1,13 @@
+import { instantiateAll } from '../../modules/util/atomic-helpers';
+import Expandable from 'cf-expandables/src/Expandable';
+import Multiselect from '../../molecules/Multiselect';
+
+const multiSelectsSelector = `.${ Multiselect.BASE_CLASS }`;
+
+if ( document.querySelector( multiSelectsSelector ) ) {
+  instantiateAll( multiSelectsSelector, Multiselect );
+}
+
+Expandable.init();
+
+
```

Comment:
TRIVIAL: Remove trailing newlines.

Reply:
Updated in https://github.com/cfpb/cfgov-refresh/pull/5207/commits/7371cd90fac1928b166a496101f3bb76ed858b96